build/jQueryShim.js

```diff
@@ -65,6 +65,11 @@ var xhr = function xhr() {
 
 var ajax = function ajax(options) {
   var request = xhr();
+
+  if (options.xhrFields && options.xhrFields.withCredentials) {
```

Comment:
is it possible for options to be null?

Reply:
no 🙂 the options contains all of the info about the connection including the connection url, so if it was null, the connection would fail regardless